### DISCUSSION TITLE
EPS to very large size PNG, not working

### DISCUSSION THREAD

**Discussion by SanjayChaudhary02** [ImageMagick/ImageMagick#1992] [General]
I have to convert EPS to very large size PNG like: 15000x15000px or more, while i am running convert command : `convert -colorspace rgb -density 300 srcFile.eps -units PixelsPerInch -resize 15000x15000 output.png`  system get hanged and very large size(>4gb) dump file creating in temp folder.
I want to know is there any limitation from Imagemagic or if any system dependency to create that much size file?
Any other available solution suggestion also be helpful.
Thanks

**Comment by snibgo** [comment]
What versions of IM and Ghostscript? On what platform?

How large is the image before "-resize"? How much free memory do you have? How much free disk space?

**Reply by SanjayChaudhary02** [reply]
IM version: ImageMagick 7.0.8-23 Q16 x64 2019-01-02,
Ghostscript version : 9.26,
image original size: 2592x1728,
free memory: 2GB,
free disk: 5GB

**Comment by urban-warrior** [comment]
See https://imagemagick.org/script/architecture.php#tera-pixel, then try this command:
```
magick -monitor -limit memory 2GiB -limit map 4GiB -define registry:temporary-path=./ \
  -colorspace rgb -density 300 srcFile.eps -units PixelsPerInch -resize 15000x15000 output.png
```
